### ISSUE TITLE
`-Zhelp` inconsistent with other unstable options

### ISSUE THREAD

**Issue by omnv** [tectonic-typesetting/tectonic#1064]
Every other unstable option is separated by a space delimiter, but `-Z help` would fail,
and only works properly when issued `-Zhelp`.

Related: https://github.com/tectonic-typesetting/tectonic/pull/929

**Comment by pkgw** [comment]
Sorry for the long delay! This should have been fixed in #1084.

**Comment by Neved4** [comment]
Minor issue I noticed recently, but it might be good to tweak it for consistency's sake.

When running without a space delimiter:
```sh
$ tectonic -Zhelp
Available unstable options:
    -Z help                     Lists all unstable options
```

When running with a space delimiter:
```sh
$ tectonic -Z help

error: The following required arguments were not provided:
    <input>
```

When passing a `foo` extra argument it works:
```sh
$ tectonic -Z help foo

Available unstable options:
    -Z help                     Lists all unstable options
```

Also in `tectonic --help`:
```sh
-Z <option>...                      Unstable options. Pass -Zhelp to show a list
```

Should be `-Z help`.

Ultimately, the behaviour without space should be kept, otherwise it'd break backwards compatibility.
